concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -203,7 +203,7 @@ resources:
     source:
       bucket: {{state_bucket}}
       region_name: {{aws_region}}
-      key: release-version
+      key: {{release_version}}
```

Comment:
What is the issue with keeping simple "release-version" as key name? The semver resource lives in the state bucket and it is not shared with the other environments.